### PR TITLE
Added instructions to run dojot on VirtualBox.

### PR DESCRIPTION
**Pull request by rascaraficci**

### CODE REVIEW

File: source/vms/vbox/guide.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+.. image:: image3.png
+        :width: 95%
+        :align: center
+
+Click on Create.
+
+Next, click on Settings/Network and check whether the network adapter is enabled and set it to 
+``Host-only``. This will allow host and guest to communicate to each other. 
+Write down the MAC Address, you will need it later.
```
Comment (giovannicuriel) [inline source/vms/vbox/guide.rst]: Why do we need to write down the MAC address?

Comment (giovannicuriel) [reply source/vms/vbox/guide.rst]: Ok, now I got it.

File: source/vms/vbox/guide.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+.. code-block:: bash
+
+	$ sudo ip address | grep -B1 -i "<YOUR MAC ADDRESS>" | head -1 | cut -d':' -f2
+
+.. image:: image5.png
+        :width: 95%
+        :align: center
+
+Edit the file ``/etc/network/interfaces``, adding ::
```
Comment (giovannicuriel) [inline source/vms/vbox/guide.rst]: The guest doesn't have default names for its network interfaces?

File: source/vms/vbox/guide.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Up to date information and installation procedures can be found at the 
+project’s documentation: 
+
+https://www.virtualbox.org/
+
+Then, you need to download a virtual machine image with dojot, which is available at: 
+
+http://dojot-iso.s3.amazonaws.com/imagem/dojot.0.1.0-dojot.vdi
+
+The next step is to create a virtual machine on VirtualBox. Chick on the New button,
```
Comment (mmagr) [inline source/vms/vbox/guide.rst]: ... machine on VirtualBox. ~Chick~ **Click** on the New button,

File: source/vms/vbox/guide.rst (hunk 2 above)
Comment (mmagr) [reply source/vms/vbox/guide.rst]: The vm is based on debian.
It's been a while now that debian changed the [naminig scheme](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/) of network interfaces to attempt to produce more hardware- stable names for network interfaces. The down side is that it uses hardware slot/bus info when naming interfaces.

**Pull request merged**